components/ide/jetbrains/backend-plugin/src/main/kotlin/io/gitpod/jetbrains/remote/stable/GitpodTerminalService.kt

```diff
@@ -1,96 +0,0 @@
-// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
-// Licensed under the GNU Affero General Public License (AGPL).
-// See License-AGPL.txt in the project root for license information.
-
-package io.gitpod.jetbrains.remote.stable
-
-import com.intellij.openapi.Disposable
-import com.intellij.openapi.application.runInEdt
-import com.intellij.openapi.project.Project
-import com.intellij.openapi.wm.ToolWindowManager
-import com.intellij.openapi.wm.ex.ToolWindowManagerListener
-import com.intellij.remoteDev.util.onTerminationOrNow
-import com.intellij.util.application
-import com.jetbrains.rd.util.lifetime.Lifetime
-import com.jetbrains.rdserver.terminal.BackendTerminalManager
-import com.jetbrains.rdserver.unattendedHost.UnattendedHostManager
-import kotlinx.coroutines.DelicateCoroutinesApi
-import kotlinx.coroutines.GlobalScope
-import kotlinx.coroutines.delay
-import kotlinx.coroutines.future.await
-import kotlinx.coroutines.launch
-import org.jetbrains.plugins.terminal.ShellTerminalWidget
-import org.jetbrains.plugins.terminal.TerminalToolWindowFactory
-import org.jetbrains.plugins.terminal.TerminalView
-import java.util.concurrent.CompletableFuture
-
-@Suppress("UnstableApiUsage", "EXPERIMENTAL_IS_NOT_ENABLED")
-@OptIn(DelicateCoroutinesApi::class)
-class GitpodTerminalService(private val project: Project) : Disposable {
```

Comment:
Ah, I just remembered one more thing:
We need to also delete this line from `resources-stable/META-INF/extensions.xml` because we deleted the file:

https://github.com/gitpod-io/gitpod/blob/49c62577081f9e8fd08a8ef36d610ecefa53f088/components/ide/jetbrains/backend-plugin/src/main/resources-stable/META-INF/extensions.xml#L8

Reply:
Updated, thanks!